src/main/java/core/basesyntax/operations/Operation.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.operations;
+
+public enum Operation {
+    b, s, p, r
```

Comment:
better to have full names as enum and add value for each of them that will be b, s ...

Reply:
done